yfinance/base.py

```diff
@@ -187,6 +186,18 @@ def history(self, period="1mo", interval="1d",
                 proxy = proxy["https"]
             proxy = {"https": proxy}
 
+        #if the ticker is MUTUALFUND or ETF, then get capitalGains events
+        self._get_info(proxy)
+        data = self._info
+        is_capital_gains_data_supported = False
```

Comment:
`is_capital_gains_data_supported` is not necessary, can be removed.

Reply:
Sure